CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 3.10.2 FATAL_ERROR)
 #============================================================================
 # Initialize the project
 #============================================================================
-project(ignition-gazebo3 VERSION 3.12.0)
+project(ignition-gazebo3 VERSION 3.13.0)
```

Comment:
@j-rivero , I think it's cleaner if we do this in a separate PR and update the changelog as usual. It makes it easier to track the releases and then announce them, and also gives visibility to the team. How about:

1. Reverting the version changes in this PR and merge it
2. Opening a new PR for the stable release, with a changelog update including all the recent changes: https://github.com/ignitionrobotics/ign-gazebo/compare/ignition-gazebo3_3.12.0...ign-gazebo3
3. Making a stable release